views/posts/index.hbs

```diff
@@ -13,7 +13,7 @@
 
                 <div class="col-1">
                   <img
-                    class="author-post-image rounded"
+                    class="author-post-image rounded" style="max-width:72px; max-height:72px"
```

Comment:
This is a nice touch. Maybe in future we use something like "author-image" for the class name for better clarity